openmls/src/group/public_group/mod.rs

```diff
@@ -396,14 +396,14 @@ impl PublicGroup {
     }
 
     /// Deletes the [`PublicGroup`] from storage.
-    pub(crate) fn delete<Storage: PublicStorageProvider>(
-        &self,
+    pub fn delete<Storage: PublicStorageProvider>(
```

Comment:
We really need integration tests for the public group. Do we have an issue for that?

Reply:
Good point. I just filed #1650.